src/components/navbar/navbar.component.tsx

```diff
@@ -94,6 +108,25 @@ const Navbar: React.FC<NavbarProps> = ({
                 >
                   <UserAvatarFilledAlt20 />
                 </HeaderGlobalAction>
+                <HeaderGlobalAction
+                  aria-label="app switcher"
+                  aria-labelledby="app swithcer Icon"
+                  name="switcher"
+                >
+                  {icon}
+                </HeaderGlobalAction>
+                <HeaderPanel
+                  aria-label="Header Panel"
+                  style={{
+                    display: switcherToggler ? "block" : "none",
+                    height: "12rem",
+                    overFlow: "overflow"
+                  }}
+                  expanded
+                >
+                  <ExtensionSlot extensionSlotName="global-nav-menu-slot"></ExtensionSlot>
```

Comment:
What's the difference between `nav-menu-slot` (which we have already) and `global-nav-menu-slot`. The former is also globally available.

Can you give a better name? What's the purpose of this panel? What kind of items should be in there?

Also please self-close empty elements for readability.

Reply:
Sir, the Items in it I'm not too sure of. So I asked Ms Grace for clarifications. Still waiting on her. But the designed showed some options already in implemented in the nav-meu-slot.